oc_cdtapi/DmsGetverAPI.py

```diff
@@ -17,26 +21,69 @@ def __init__(self, *args, **argv):
         # TODO: re-factor when Python2 support will be deprecated
         super(DmsGetverAPI, self).__init__(*args, **argv)
 
-        # delivery states in process
-        self.waiting_states = ['INITIATED', 'PROCESSING', 'QUEUED']
+        # auth token
+        self.auth_token = None
 
-        # delivery states finished
-        self.exit_states = ['READY', 'FAILED']
+        # delivery states or images in process
+        self.waiting_states = ['INITIATED', 'PROCESSING', 'QUEUED', 'UNKNOWN']
+
+        # delivery states or images finished
+        self.exit_states = ['READY', 'FAILED', 'SUCCESS']
 
         # wait for state timeout
         self.wait_state_timeout = 1000
 
         # wait for state request interval
         self.wait_state_sleep = 30
 
+        # oracle version
+        self.oracle_version = '19.16.0.0.0'
+
+        # oracle encoding
+        self.oracle_encoding = 'AL32UTF8'
+
+        # oracle edition
+        self.oracle_edition = 'EE'
+
+    def create_custom_image(self, version=None, distr_type=None, client_filter=None, client_code=None):
+        """
+        Sends request to create custom image
+        :param str version: version of product
+        :param str distr_type: type of product
+        :param str client_filter: client specific filter list
+        :param str client_code: client code
+        """
+        logging.debug('Reached create_custom_image')
+        logging.debug('version: [%s]' % version)
+        logging.debug('distr_type: [%s]' % distr_type)
+        logging.debug('client_filter: [%s]' % client_filter)
+        logging.debug('client_code: [%s]' % client_code)
+
+        self.web.auth = None
+        self.raise_exception_high = 499
```

Comment:
It is a bad idea to drop these fundamentals duiing one call and not restore it afterwards.